src/patternfly/components/BackToTop/back-to-top.scss

```diff
@@ -1,21 +1,23 @@
 // @debug $back-to-top; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$back-to-top} {
-  --#{$back-to-top}--Right: var(--#{$pf-global}--spacer--2xl);
-  --#{$back-to-top}--Bottom: var(--#{$pf-global}--spacer--lg);
-  --#{$back-to-top}--md--Bottom: var(--#{$pf-global}--spacer--2xl);
-  --#{$back-to-top}--c-button--FontSize: var(--#{$pf-global}--FontSize--xs);
-  --#{$back-to-top}--c-button--BorderRadius: var(--#{$pf-global}--BorderRadius--lg);
-  --#{$back-to-top}--c-button--PaddingTop: var(--#{$pf-global}--spacer--xs);
-  --#{$back-to-top}--c-button--PaddingRight: var(--#{$pf-global}--spacer--sm);
-  --#{$back-to-top}--c-button--PaddingBottom: var(--#{$pf-global}--spacer--xs);
-  --#{$back-to-top}--c-button--PaddingLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$back-to-top}--c-button--BoxShadow: var(--#{$pf-global}--BoxShadow--lg-bottom);
+:root {
+  --#{$back-to-top}--Right: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--Bottom: var(--pf-t--global--spacer--lg);
+  --#{$back-to-top}--md--Bottom: var(--pf-t--global--spacer--2xl);
+  --#{$back-to-top}--c-button--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$back-to-top}--c-button--BorderRadius: var(--pf-t--global--border--radius--large);
```

Comment:
This uses the pill radius in design, but I think we can just remove this in favor of the pill radius on the button. I imagine the distinction between this button and the button component is just the font-size, padding, and box-shadow.